packages/tools/package.json

```diff
@@ -10,12 +10,12 @@
     "chalk": "^3.0.0",
     "lodash": "^4.17.15",
     "mime": "^2.4.1",
-    "node-fetch": "^2.5.0"
+    "node-fetch": "^2.6.0"
```

Comment:
I've bumped it because of https://github.com/node-fetch/node-fetch/blob/master/CHANGELOG.md#v260 :

> Fix: incorrect Content-Length was returned for stream body in 2.5.0 release; note that node-fetch doesn't calculate content length for stream body.